lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
We need not just the set up but a test workflow so that we know how to see
all the screens.

On Wed, May 20, 2015 at 10:20 AM, Marko Jevtić <notifications@github.com>
wrote:

> In lms/static/sass/views/_shoppingcart.scss
> <https://github.com/edx/edx-platform/pull/7987#discussion_r30706347>:
>
> > @@ -1,336 +1,194 @@
> >  // lms - views - shopping cart
> >  // ====================
> >
> > +$enrollment-details-width-difference: $baseline*11.85;
>
> @martynjames <https://github.com/martynjames> agreed to help with setting
> it up, since I'm having trouble SSHing into it
>
> Will let you know as soon as it's up
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7987/files#r30706347>.
>